functions.php

```diff
@@ -54,18 +60,83 @@ function createBottlesHtml(array $allBottles): string {
+        $bottlesHtml .=
+            '<div class="bottleCard">
+            <a class="anchor" id="dbId-'.$bottle['id'].'"></a>';
+        if(isset($editCardId) && $editCardId == $bottle['id']) {
+            $bottlesHtml .='
+                <form class="card-edit-button-form" method="POST" action="index.php#dbId-'.$bottle['id'].'">
+                    <div class="card-outer">
+                        <h3>Bottle number '.$bottle['id'].'</h3>
+                        <div class="card-inner">
+                            <div>
+                                <label for="item-name">Bottle name</label>
+                                <input type="text" id="item-name" name="item-name" value="'.$bottle['itemname'].'" required>
+                            </div>
+                            <div>
+                                <label for="type">Type of alcohol</label>
+                                <select id="type" name="type" required>
+                                    <option selected hidden value="'.$bottle['type'].'">'.$bottle['type'].'</option>
+                                    <option value="Rum">Rum</option>
+                                    <option value="Rye">Rye</option>
+                                    <option value="Vodka">Vodka</option>
+                                    <option value="Whisky">Whisky</option>
+                                </select>
+                            </div>
+                        </div>
+                        <div class="card-inner">
+                            <div>
+                                <label for="purchase-location">Purchase location</label>
+                                <input type="text" id="purchase-location" name="purchase-location"  value="'.$bottle['purchaselocation'].'" required>
+                            </div>
+                            <div>
+                                <label for="purchase-date">Purchased date</label>
+                                <input type="date" id="purchase-date" name="purchase-date" value="'.$bottle['purchasedate'].'" required>
+                            </div>
+                        </div>
+                    </div>
+                    <input type="hidden" value="'.$bottle['id'].'" name="id">
+                    <input type="submit" value="Update item" class="form-submit-button">
+                    <p class="error-message"><?=$errorMessage?></p>
+                </form>';
+        } else {
+            $bottlesHtml .='
+                <div class="card-parent">
+                    <div>
+                        <h3>Bottle number '.$bottle['id'].'</h3>
+                        <form class="edit-initiate-form" method="GET" action="index.php#dbId-'.$bottle['id'].'">
+                            <input type="hidden" name="editCardId" value="'.$bottle['id'].'">
+                            <input type="image" alt="edit-button" src="edit.png" class="edit-button-on-card">
+                        </form>
```

Comment:
instead of all this, just use `<a>` with the appropriate query params (as discussed)

Reply:
unfortunately that doesn't force a page refresh